### PR TITLE
Move to GitHub merge queues.

### PR DESCRIPTION
**Pull request by ltratt**

### CODE REVIEW

File: .github/workflows/sdci.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  merge_group:
+
+# This is required to silence emails about the workflow having no jobs.
+# We simply define a dummy job that does nothing much.
+jobs:
+  dummy:
+    runs-on: ubuntu-latest
+    steps:
+      - run: /usr/bin/true
+    cd ..
```
Comment (vext01) [inline .github/workflows/sdci.yml]: what's this line?

Comment (ltratt) [reply .github/workflows/sdci.yml]: Oops. Fixed in e552af2.

**Comment by vext01** [comment]
Sorry, please squash.

**Comment by ltratt** [comment]
Squashed.

**Pull request merged**